mycodes/util/utils.go

```diff
@@ -0,0 +1,68 @@
+package util
```

Comment:
you don't need a separate package for this, in the main package, all _test.go files can share code between each other.